docs/topics/commonly_used_plugins.rst

```diff
@@ -10,284 +10,69 @@ Some commonly-used plugins
     you are upgrading from a previous version. Please refer to
     :ref:`Upgrading from previous versions <upgrade-to-3.0>`
 
-These are the recommended plugins to use with django CMS.
 
-..  module:: djangocms_file
+Please note that dozens if not hundreds of different django CMS plugins have been made made available under open-source
+licences. Some, like the ones on this page, are like to be of general interest, while others are highly specialised.
```

Comment:
This line "Some, like the ones on this page, are like to be of general interest" is a bit confusing, not sure what you mean here.